Atualização dos Textos da Tela Perguntas Frequentes

Alines009 opened the pull request:
Primeira atualização das perguntas e respostas da tela "Perguntas Frequentes". As informações estão em inglês e português. 

Edit: Relacionado a issue #19.

Alines009 committed:
Merge pull request #1 from rafaeldcampbell/develop

Update Fork

morganadcampbell committed:
Removendo tela de login

Alines009 committed:
Merge pull request #2 from rafaeldcampbell/develop

Update Fork

morganadcampbell committed:
Reestruturando arvore de diretórios do projeto

morganadcampbell committed:
Adicionando teste para a inicialização do aplicativo

Alines009 committed:
Atualização dos Textos da tela Perguntas Frequentes

Primeira versão das informações da tela "Perguntas Frequentes" em português e em inglês

Alines009 committed:
Conserto nas páginas de internacionalização

Remoção de vígulas que causavam erro

morganadcampbell reviewed (changes requested):
Um dos testes falhou. Favor:

- [x] Alterar os testes, em especial o "Testando ask (ingles)"
- [x] Executar os testes (use `flutter test`)

Alines009 committed:
Validação dos Testes de Internacionalização

Atualização do arquivo de testes de acordo com a ultima versão dos textos de internacionalização.

morganadcampbell committed:
Merge branch 'develop' into feature2

morganadcampbell reviewed (approved):
Testes aprovados.
